config.json

```diff
@@ -76,7 +76,8 @@
     "connect",
     "ledger",
     "poker",
-    "variable-length-quantity"
+    "variable-length-quantity",
+    "all-your-base"
```

Comment:
are we OK with this placement? all-your-base might not be as involved as, say, connect and poker, maybe it could go before those at least?

(I guess I also think VLQ is placed a bit late, but that can be considered in a place that's not this PR)